frontend/test/parameters/parameters.integ.spec.js

```diff
@@ -60,8 +60,8 @@ const getRelativeUrlWithoutHash = (url) =>
     url.replace(/#.*$/, "").replace(/http:\/\/.*?\//, "/")
 
 const COUNT_ALL = "200";
-const COUNT_DOOHICKEY = "56";
-const COUNT_GADGET = "43";
+const COUNT_DOOHICKEY = "51";
```

Comment:
I'm assuming these values changed cause of the update to the Sample Dataset? Is there any way to avoid this being so fragile and causing test failures?

Reply:
yep, they did. we could simply change the lines where those variables are used to be less specific; they could verify that the scalar has a value of >= 0 instead of checking the exact value

Reply:
I've started doing that, but the problem soon becomes that sometimes (eg. when testing binning) you want to test for particular values. Some can probably be worked around by changing queries. 

The current approach is not ideal especially as the sample dataset will probably change more in the coming months, but fixing this probably deserves it's own PR as it's not a trivial change.